source/llvm/build-source-tarball.sh

```diff
@@ -68,6 +68,7 @@ function build_llvm() {
   wrap cmake ${THIS_DIR}/$LLVM.src \
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_INSTALL_PREFIX=$LOCAL_INSTALL \
+      -DGCC_INSTALL_PREFIX=$GCC_INSTALL_PREFIX \
```

Comment:
If it works with all of the versions of LLVM that we build, then it would be a good idea. It looks like LLVM 3.3 didn't use cmake